### PR TITLE
Work aboutus feature

### PR DESCRIPTION
**Pull request by rychrr**
Hi, 

I have made updates to the  mobile version page and template;

- Included the work section 
- included the about me section 
- Updated the CSS to follow DRY principle 
- Applied grid display method in building the work section
- updated the readme file

Regards,

### CODE REVIEW

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @rychrr   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter error :heavy_check_mark: 
- Well-detailed PR summary :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] Kindly make sure to remove the horizontal scroll on your website because it doesn't look professional.

    _**Suggestion:
       ` body {
          padding: 0;
          margin: 0;
        }`
**_

   ### YOUR APP BEHAVIOUR
   chrome-capture-2023-3-29 (1)


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -118,3 +120,277 @@
...
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  padding: 12px;
+  background: #ff6b00;
+  color: #fff;
+}
```
[the remark this answers is not in the document]
Comment (rychrr) [reply style.css:223]: Thanks for the feedback, 
This has been resolved:

**Comment by rychrr** [comment]
Thank you for the review and feedback.
All updates were applied

Regards,

**Review by TadesseDev — APPROVED** [review]
### Hi @rychrr 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rychrr** [comment]
Thanks @TadesseDev @levy002

**Pull request merged**